sanic/config.py

```diff
@@ -135,23 +160,46 @@ def load_environment_vars(self, prefix=SANIC_PREFIX):
                         self[config_key] = float(v)
                     except ValueError:
                         try:
-                            self[config_key] = strtobool(v)
+                            self[config_key] = str_to_bool(v)
                         except ValueError:
                             self[config_key] = v
 
 
-def strtobool(val):
-    """
-    This function was borrowed from distutils.utils. While distutils
-    is part of stdlib, it feels odd to use distutils in main application code.
-
-    The function was modified to walk its talk and actually return bool
-    and not int.
-    """
-    val = val.lower()
-    if val in ("y", "yes", "t", "true", "on", "1"):
-        return True
-    elif val in ("n", "no", "f", "false", "off", "0"):
-        return False
-    else:
-        raise ValueError("invalid truth value %r" % (val,))
+    def update_config(self, config: Union[bytes, str, dict, Any]):
+    """Update app.config.  
+    
+    Note:: only upper case settings are considered.  
+    
+    You can upload app config by providing path to py file holding settings.  
+    
+        # /some/py/file  
+        A = 1  
+        B = 2  
+    
+        config.update_config("${some}/py/file")  
+    
+    Yes you can put environment variable here, but they must be provided in format: ${some_env_var},  
+    and mark that $some_env_var is treated as plain string.  
+    
+    You can upload app config by providing dict holding settings.  
+    
+        d = {"A": 1, "B": 2}  
+        config.update_config(d)  
+    
+    You can upload app config by providing any object holding settings,  
+    but in such case config.__dict__ will be used as dict holding settings.  
+    
+        class C:  
+            A = 1  
+            B = 2  
+        config.update_config(c)"""
```

Comment:
Should this be `config.update_config(c)` or `config.update_config(C)`?

Reply:
Right :)